.eslintrc

```diff
@@ -2,10 +2,12 @@
   "extends": "@beyondessential/js",
   "parser": "babel-eslint",
   "rules": {
+    "global-require": "off",
```

Comment:
I can see that this is deprecated in eslint 7, is that the reason for turning it off? I believe we're in a good habit with this across the board anyway, so I'm not worried, just interested in the reason.

Reply:
Good point, I should have left a comment!

We generally use ES6 modules in this repo, so `require` is mostly used in scripts like the webpack/create-react-app ones. In those instances, there are justified cases where we want to conditionally require a dependency in the body of the script.

Also, the [dynamic import proposal](https://github.com/tc39/proposal-dynamic-import) is in Stage 4, and I expect that once this is included in ES it may change a bit how we handle imports. In conclusion, and taking into account that this rule is deprecated, I thought we may as well as turn it off for this repository. I believe we should still keep it in the base style guide, since other repos may rely heavily in CommonJS modules.